msm/src/column_env.rs

```diff
@@ -38,7 +38,9 @@ impl<'a, F: FftField> TColumnEnvironment<'a, F> for ColumnEnvironment<'a, F> {
     ) -> Option<&'a Evaluations<F, Radix2EvaluationDomain<F>>> {
         let witness_columns_n: usize = self.witness.len();
         let coefficients_columns_n: usize = self.coefficients.len();
-        let crate::columns::Column::X(i) = col;
+        let crate::columns::Column::X(i) = col else {
```

Comment:
This means you cannot get a lookup column with this function?

Reply:
This is not implemented yet. See in the https://github.com/o1-labs/proof-systems/pull/1980